src/qmapshack/gis/CGisWorkspace.h

```diff
@@ -309,6 +309,16 @@ class CGisWorkspace : public QWidget, private Ui::IGisWorkspace
     void getItemsByKeys(const QList<IGisItem::key_t>& keys, QList<IGisItem*>& items);
 
     void getNogoAreas(QList<IGisItem*>& nogos);
+
+    /**
+           @brief Get access to the workspace list
+
+           @return the workspace list
+    */
+    CGisListWks *getWks(void)
+    {
+        return treeWks;
+    }
```

Comment:
There is no reason to make a private class member public for sure.

Reply:
The function is removed with [QMS-396] Modified TrkToRtDialog